helm-charts/falcon-sensor/questions.yaml

```diff
@@ -1,96 +1,98 @@
-labels:
-      io.cattle.role: cluster # options are cluster/project
-      io.rancher.app_min_version: 7.3.8
-categories:
-
 questions:
-    - variable: image.repository
-      description: "URL of container image repository holding containerized Falcon sensor. Defaults to 'falcon-sensor'."
-      required: true
-      type: string
-      default: falcon-sensor
-      label: Container Image Repository
-      group: "Required Configuration Settings"
+  - variable: node.image.repository
+    description: "URL of container image repository holding containerized Falcon sensor. Defaults to 'falcon-sensor'."
+    required: true
+    type: string
+    default: falcon-node-sensor
+    label: Container Image Repository
+    group: "Node Container Images"
```

Comment:
```suggestion
  - variable: node.image.repository
    description: "URL of container image repository holding containerized Falcon sensor. Defaults to 'falcon-sensor'."
    required: true
    type: string
    default: falcon-sensor
    label: Container Image Repository
    group: "Node Container Images"
```

Reply:
Discrepency between documented default as ``falcon-sensor`` and the ``default: falcon-node-sensor`` value

Reply:
huh.... thought I fixed that.

Reply:
fixed